config/fgcmBuildStarsTable.py

```diff
@@ -1,10 +1,5 @@
-"""
-HSC-specific overrides for FgcmBuildStarsTable
-"""
-
```

Comment:
Ditto

Reply:
As above, these comments are now largely redundant, but keen to hear your thoughts.

Reply:
All now removed.